packages/prosess-aarskvantum-oms/package.json

```diff
@@ -11,17 +11,28 @@
     "@fpsak-frontend/shared-components": "1.0.0",
     "@fpsak-frontend/utils": "1.0.0",
     "@k9-sak-web/types": "1.0.0",
+    "classnames": "^2.2.6",
+    "lodash.throttle": "^4.1.1",
     "moment": "^2.24.0",
     "nav-frontend-alertstriper": "3.0.23",
     "nav-frontend-alertstriper-style": "2.0.17",
-    "nav-frontend-core": "4.0.12",
+    "nav-frontend-core": "^5.0.10",
     "nav-frontend-grid": "1.0.32",
+    "nav-frontend-hjelpetekst": "^2.0.47",
+    "nav-frontend-hjelpetekst-style": "^2.0.36",
+    "nav-frontend-ikoner-assets": "^2.0.8",
+    "nav-frontend-js-utils": "^1.0.15",
+    "nav-frontend-lenker": "^1.0.53",
+    "nav-frontend-lenker-style": "^0.2.39",
     "nav-frontend-paneler": "^2.0.10",
+    "nav-frontend-popover": "^0.0.41",
+    "nav-frontend-popover-style": "^0.0.13",
     "nav-frontend-tabs": "^0.0.38",
     "nav-frontend-typografi": "2.0.26",
-    "nav-frontend-typografi-style": "1.0.22",
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1",
+    "nav-frontend-typografi-style": "^1.0.31",
+    "prop-types": "^15.7.2",
+    "react": "^17.0.1",
```

Comment:
Det gjelder spesielt for denne. Denne vil også gi en del warnings da nav-frontend-moduler ikke støtter react 17 ennå